packages/fast-jss-utilities/src/color.ts

```diff
@@ -0,0 +1,13 @@
+export function convertHexToRgb(hex: string, alpha?: number): string {
```

Comment:
nit: Just a thought - why not just `hexToRgb` I don't think any understanding is lost and it is shorter.